soc/arm/nuvoton_npcx/common/soc_dt.h

```diff
@@ -181,10 +162,10 @@
  * @param inst instance number for compatible defined in DT_DRV_COMPAT.
  * @return an array of npcx_alt items.
  */
-#define NPCX_DT_ALT_ITEMS_LIST(inst) {             \
-	UTIL_LISTIFY(NPCX_DT_ALT_ITEMS_LEN(inst),  \
-		     NPCX_DT_ALT_ITEMS_FUNC,       \
-		     inst)                         \
+#define NPCX_DT_ALT_ITEMS_LIST(inst) {			\
+	UTIL_LISTIFY(DT_NUM_PINCTRLS_BY_IDX(inst, 0),	\
```

Comment:
Because the parameter is "inst", it should use DT_DRV_INST equivalent macro here.
`
UTIL_LISTIFY(DT_INST_NUM_PINCTRLS_BY_IDX(inst, 0),      \
`

Reply:
Thanks @ChiHuaL; now fixed.